src/core/manager-serialize.c

```diff
@@ -376,9 +376,9 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                 } else if ((val = startswith(l, "log-level-override="))) {
                         int level;
 
-                        level = log_level_from_string(val);
-                        if (level < 0)
-                                log_notice("Failed to parse log-level-override value '%s', ignoring.", val);
+                        r = safe_atoi(val, &level);
```

Comment:
Hmm? This doesn't seem necessary since log_level string table accepts numbers via DEFINE_STRING_TABLE_LOOKUP_WITH_FALLBACK?

Reply:
Indeed.